packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -58,6 +58,20 @@ export class WaferMap extends FoundationElement {
         colors: [],
         values: []
     };
+
+    @observable public renderReady = false;
+
+    public override connectedCallback(): void {
+        super.connectedCallback();
+        // The component has been attached to the DOM and the attributes, properties are available
+        // Use this for initialization/initial render
+
+        // Simulate the time required until render is complete, before displaying the waferMap
+        // User this for conditional rendering
+        setTimeout((): void => {
+            this.renderReady = true;
+        }, 2000);
```

Comment:
I do not think we should go down the path with renderReady. It does not actually represent the expected behavior of the control which should be responsive to attribute changes. The pattern of which was discussed in this thread of the spec: https://github.com/ni/nimble/pull/805#discussion_r1018156031

I'm also concerned because the other open PR is also initializing behavior in a manner that does not align with the spec as discussed here: https://github.com/ni/nimble/pull/835#discussion_r1034205301

If we start introducing code and workarounds for behavior that is not aligned with what the control is speced to actually have we are introducing complexity and tech debt for little benefit. I do not believe the cost is high to respond to the attribute changes properly as discussed in the spec so I'm going to mark the PR as waiting on that behavior change unless there is more discussion needed.

Reply:
> It does not actually represent the expected behavior of the control which should be responsive to attribute changes. The pattern of which was discussed in this thread of the spec: #805 (comment)
>
> I'm also concerned because the other open PR is also initializing behavior in a manner that does not align with the spec as discussed here: #835 (comment)

Can you comment on when that pattern will be implemented? Add discussed I'm concerned the longer we wait the more tech debt we introduce (and I have to comment on it in every pr)

Reply:
I think that we, most likely, will get to this implementation once we will have the rendering method for the wafermap and the dies - i.e. code that takes some actual data and plots it. This part is still in the prototyping phase ... I cannot give you an exact time, but I would image sometime the end of next week (starting from tomorrow we are on vacation until then end of the current week)

Reply:
> will get to this implementation once we will have the rendering method for the wafermap and the dies

I'd strongly recommend doing it first. It's not blocked on the rendering behavior chosen but would likely influence the implementation. I won't block this PR on it.

Reply:
I think that we are on the same page with this - we will add a basic implementation of this pattern as soon as possible